fixtures/pytest_store.py

```diff
@@ -74,6 +74,10 @@ def __init__(self):
         self._user = None
 
     @property
+    def has_config(self):
+        return self.config is not None
```

Comment:
If store.config is None, then store doesn't have a config. We don't need a method for this.